### PR TITLE
PLT-2252 Update push notifications

### PR DESCRIPTION
**Pull request by jwilander**
Note the Jenkins build will always fail since it's using the new build system. Travis should pass, however.

### CODE REVIEW

File: api/post.go [hunk 1]
```diff
@@ -663,38 +663,43 @@ func sendNotifications(c *Context, post *model.Post, team *model.Team, channel *
...
-							if channel.Type == model.CHANNEL_DIRECT {
-								msg.Message = senderName + userLocale("api.post.send_notifications_and_forget.push_message")
-							} else {
-								msg.Message = senderName + userLocale("api.post.send_notifications_and_forget.push_mention") + channelName
-							}
+								if channel.Type == model.CHANNEL_DIRECT {
+									msg.Message = senderName + userLocale("api.post.send_notifications_and_forget.push_message")
+								} else {
+									msg.Message = senderName + userLocale("api.post.send_notifications_and_forget.push_mention") + channelName
+								}
```
Comment (enahum) [inline api/post.go]: Why Push Notification Contents was removed?

Comment (jwilander) [inline api/post.go]: It wasn't removed, that just came in after 2.1 and these changes are based off the 2.1 release branch, not master

**Comment by crspeller** [comment]
+1
@jwilander Remember that I won't be merging this branch back into master, so you will have to submit these changes there too.

**Comment by jwilander** [comment]
Yes, I'll submit a PR for that

**Comment by hmhealey** [comment]
+1

**Pull request merged**